lib/chef-dk/command/generator_commands/cookbook.rb

```diff
@@ -96,7 +96,13 @@ def emit_post_create_message
           if have_delivery_config?
             msg("Your cookbook is ready. To setup the pipeline, type `cd #{cookbook_name_or_path}`, then run `delivery init`")
           else
-            msg("Your cookbook is ready. Type `cd #{cookbook_name_or_path}` to start working.")
+            msg("Your cookbook is ready. Type `cd #{cookbook_name_or_path}` to enter it.")
+            msg("\nThere are several commands you can run to get started locally developing and testing your cookbook.")
+            msg("Type `delivery local --help` to see a full list.")
+            msg("\nWhy not start by writing a test? Tests for the default recipe are stored at:\n")
+            msg("test/integration/default/default_spec.rb")
+            msg("\nIf you'd prefer to dive right in, the default recipe can be found at:")
+            msg("\nrecipes/default.rb\n")
```

Comment:
Eh, it was intended to be a jumping off point for getting started with delivery, but we weren't 100% on what the best approach was.